opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/point.py

```diff
@@ -184,16 +319,24 @@ class Metric:
     unit: Optional[str]
     data: DataT
 
+    def to_dict(self) -> MetricDict:
+        return {
+            "name": self.name,
+            "description": self.description or "",
+            "unit": self.unit or "",
+            "data": self.data.to_dict(),
+        }
+
     def to_json(self, indent=4) -> str:
-        return dumps(
-            {
-                "name": self.name,
-                "description": self.description or "",
-                "unit": self.unit or "",
-                "data": loads(self.data.to_json(indent=indent)),
-            },
-            indent=indent,
-        )
+        return dumps(self.to_dict(), indent=indent)
+
+
+class ScopeMetricsDict(typing.TypedDict):
+    """Dictionary representation of a ScopeMetrics object."""
+
+    scope: typing.Any
```

Comment:
Would it be better to use the dictionary type here instead of `typing.Any` here and on the next line?